pkg/controllers/status/work_status_controller.go

```diff
@@ -294,6 +294,12 @@ func (c *WorkStatusController) handleDeleteEvent(ctx context.Context, key keys.F
 		return nil
 	}
 
+	// skip processing as the work object is suspended for dispatching.
+	if util.IsWorkSuspendDispatching(work) {
```

Comment:
In this PR, there is no need to move the position of the `IsWorkSuspendDispatching` function; we can directly modified the package name.